resources/public/api/conf/2.0/employments_financial_details_retrieve.yaml

```diff
@@ -14,6 +14,7 @@ description: |
   | NOT_FOUND                                | Simulates the scenario where no data is found.                                                                                                                                                                                             |
   | STATEFUL                                 | Performs a stateful retrieve. <p> Information and data held by HMRC is not available in the sandbox environment. Providing ‘source=hmrcHeld’ will result in the expected HTTP 404 (matching resource not found) error being returned. </p> |
   | REQUEST_CANNOT_BE_FULFILLED              | Simulates the scenario where the request cannot be fulfilled for various reasons other than the ones triggering other errors.                                                                                                              |
+  | OUTSIDE_AMENDMENT_WINDOW                 | Simulates the scenario where the request cannot be completed as it is outside the amendment window.                                                                                                                                        |
```

Comment:
This is the wrong end point. Looks like it was missed for Amend a Custom employment